kubetest/main.go

```diff
@@ -654,6 +654,16 @@ func prepareGcp(o *options) error {
 		return err
 	}
 	if o.provider == "gce" {
+		// To avoid failing large tests due to some flakes in starting nodes,
+		// allow for small percentage of nodes to not start during cluster startup.
+		if numNodes, err := strconv.Atoi(os.Getenv("NUM_NODES")); err == nil {
```

Comment:
Instead of this, can't you instead take numNodes from `--gcp-nodes` flag instead?
Though before that, we need to move all our jobs from using NUM_NODES to --gcp-nodes (this will also help cleanup one more env var from those files).

Reply:
That isn't easier.
You need parsing anyway (because "--gcp-nodes" is string flag anyway).
And in MigrateOptions, we copy the value of flag to NUM_NODES env var anyway.

So if I need to operate on env vars (for setting), I prefer to use only env vars instead of mixing both env vars and flags.

Reply:
See my comment there. I still think this approach is actually better.